eng/Version.Details.xml

```diff
@@ -12,6 +12,138 @@
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>059caa278b1e5a62d0d60c9d84bdd737b04e6da8</Sha>
     </Dependency>
+     <Dependency Name="Microsoft.Extensions.Primitives" Version="6.0.2-mauipre.1.22074.5">
```

Comment:
Before merging this one, we should find out if it needs `CoherentParentDependency`.

Reply:
This is correct, we shouldn't have `CoherentParentDependency` here.